api/scpca_portal/config/spatial_readme_template.md

```diff
@@ -0,0 +1,36 @@
+# Alex's Lemonade Stand Foundation Single-cell Pediatric Cancer Atlas
+
+The [Single-cell Pediatric Cancer Atlas](https://scpca.alexslemonade.org) is a database of single-cell and single-nuclei data from pediatric cancer clinical samples and xenografts, built by the [Childhood Cancer Data Lab](https://www.ccdatalab.org/) at [Alex's Lemonade Stand Foundation](https://www.alexslemonade.org/).
+
+## Contents
+
+This download includes gene expression data from libraries processed using spatial transcriptomics and associated metadata for samples from project [{project_accession}]({project_url}) in the ScPCA portal.
+
+For all spatial transcriptomics libraries (indicated by the `SCPCL` prefix) , a `SCPCL000000_spatial` folder will be nested inside the corresponding sample folder (`SCPCS` prefix) in the download.
+See the [FAQ section about samples and libraries](https://scpca.readthedocs.io/en/latest/faq.html#what-is-the-difference-between-samples-and-libraries) for more information.
+
+Inside the `SCPCL000000_spatial` folder will be the following folders and files:
+
+- A `raw_feature_bc_matrix` folder containing the [unfiltered counts files](https://support.10xgenomics.com/spatial-gene-expression/software/pipelines/latest/output/matrices)
+- A `filtered_feature_bc_matrix` folder containing the [filtered counts files](https://support.10xgenomics.com/spatial-gene-expression/software/pipelines/latest/output/matrices)
+- A `spatial` folder containing [images and position information](https://support.10xgenomics.com/spatial-gene-expression/software/pipelines/latest/output/images)
+- A `SCPCL000000_spaceranger_summary.html` file containing the [summary html report provided by Space Ranger](https://support.10xgenomics.com/spatial-gene-expression/software/pipelines/latest/output/summary)
+- A `SCPCL000000_metadata.json` file containing library processing information.
+
+Also included in each download is a `spatial_metadata.tsv`, a tab separated values table, with one row per library and columns containing pertinent metadata corresponding to that library.
+
+See the [Downloadable files](https://scpca.readthedocs.io/en/latest/download_files.html) section in our documentation for more detailed information on files included in the download.
```

Comment:
Do we want to link directly to the spatial section of the docs? 

Which leads me to the question: why is the spatial section not showing up in https://scpca.readthedocs.io/en/latest/download_files.html ? Some of the spatial content is there for processing, but the section [here](https://github.com/AlexsLemonade/scpca-docs/blob/main/docs/download_files.md#spatial-transcriptomics-libraries) is not showing up on readthedocs.io

Reply:
Okay one option is to add it now and it will be a broken link that should work when things are deployed... or we could hold off and update it later when the docs are updated...?